tests/test_cache.py

```diff
@@ -282,3 +419,19 @@ def get_cache_expire_string_year(self):
     @Cache(key="test_cache_expire_string_year", invalidate=True)
     def clear_cache_expire_string_year(self):
         pass
+
+    @Cache(key="test_cache_empty", expire=timedelta(seconds=2))

```

Comment:
lets try this for the entire test. for the reason i want this to be a good example of how to use cache. 

all cache `key` will become a constant that will be reused also in the invalidate 

```Python
CACHE_TEST_CACHE_EMPTY = 'test_cache_empty'
```